include/cantera/zeroD/ReactorBase.h

```diff
@@ -306,6 +319,9 @@ class ReactorBase
 
     //! Composite thermo/kinetics/transport handler
     shared_ptr<Solution> m_solution;
+
+    // Data associated each sensitivity parameter
+    vector<SensitivityParameter> m_sensParams;
```

Comment:
Does promoting this variable to `ReactorBase` allow the `ReactorSurface::m_params` variable to be deleted?